wandb/apis/public/registries.py

```diff
@@ -0,0 +1,425 @@
+            "filters": json.dumps(self.filter),
+        }
+
+        super().__init__(client, variables, per_page)
+
+    def collections(self, filter: Optional[Dict[str, Any]] = None) -> "Collections":
+        return Collections(self.client, self.organization, self.filter, filter)
+
+    def versions(self, filter: Optional[Dict[str, Any]] = None) -> "Versions":
+        return Versions(
+            self.client,
+            self.organization,
+            self.filter,
+            None,
+            filter,
+        )
+
+    @property
+    def length(self):
+        if self.last_response:
+            return len(
+                self.last_response["organization"]["orgEntity"]["projects"]["edges"]
+            )
+        else:
+            return None
+
+    @property
+    def more(self):
+        if self.last_response:
+            return self.last_response["organization"]["orgEntity"]["projects"][
+                "pageInfo"
+            ]["hasNextPage"]
+        else:
+            return True
+
+    @property
+    def cursor(self):
+        if self.last_response:
+            return self.last_response["organization"]["orgEntity"]["projects"][
+                "pageInfo"
+            ]["endCursor"]
+        else:
+            return None
+
+    def convert_objects(self):
+        return [
+            Registry(
+                self.client,
+                self.organization,
+                self.last_response["organization"]["orgEntity"]["name"],
+                r["node"]["name"],
+                r["node"],
+            )
+            for r in self.last_response["organization"]["orgEntity"]["projects"][
+                "edges"
+            ]
+        ]
+
+
+class Registry(Attrs):
+    """Registry in the Global registry."""
+
+    def __init__(self, client, organization, entity, project, attrs):
+        self.client = client
+        self.full_name = project
+        self.name = self.full_name.replace("wandb-registry-", "")
+        self.entity = entity
+        self.organization = organization
+        self.description = attrs.get("description", "")
+        self.allow_all_artifact_types = attrs.get(
+            "allowAllArtifactTypesInRegistry", False
+        )
+        self.artifact_types = [
+            t["node"]["name"] for t in attrs.get("artifactTypes", {}).get("edges", [])
+        ]
+
+    @property
+    def path(self):
+        return [self.entity, self.name]
+
+    def collections(self, filter: Optional[Dict[str, Any]] = None):
+        registry_filter = {
+            "name": self.full_name,
+        }
+        return Collections(self.client, self.organization, registry_filter, filter)
+
+    def versions(self, filter: Optional[Dict[str, Any]] = None):
+        registry_filter = {
+            "name": self.full_name,
+        }
+        return Versions(self.client, self.organization, registry_filter, None, filter)
+
+
+class Collections(Paginator):
+    """Iterator that returns Artifact collections."""
```

Comment:
```suggestion
    """Iterator that returns Artifact collections in W&B Registry."""
```
(or something along those lines to differentiate this class from the ArtifactCollections class?)